doc/reverse-engineered-immoscout.md

```diff
@@ -77,4 +77,4 @@ curl -H "User-Agent: ImmoScout24_1410_30_._" \
 
 
 ## Parameters 
-The parameters between web and mobile are very different which is why we have to translate them. Please see `immoscout-web-translator.js`.
\ No newline at end of file
+The parameters between web and mobile are very different which is why we have to translate them. Please see [/lib/services/immoscout/immoscout-web-translator.js](https://github.com/orangecoding/fredy/blob/master/lib/services/immoscout/immoscout-web-translator.js).
```

Comment:
Please revery moving the doc. It is linked in numerous blogs and if we move it, we break all those links. Thanks

Reply:
Moved it back!